test-utils/src/main/java/org/hibernate/validator/testutil/DescriptorAssert.java

```diff
@@ -54,7 +55,7 @@ public void containsConversion(Class<?> from, Class<?> to) {
 			}
 
 			if ( !foundMatchingConversion ) {
-				fail( String.format( "<%s> does not contain a conversion from <%s> to <%s>.", actual, from, to ) );
+				fail( String.format( Locale.getDefault(), "<%s> does not contain a conversion from <%s> to <%s>.", actual, from, to ) );
```

Comment:
@marko-bekhta Is this still a left-over? I.e. are you planning to revert this change as per our discussion above? Same question for some more places where `Locale.getDefault()` is used now.